evolver/settings.py

```diff
@@ -12,6 +12,7 @@ class Settings(BaseSettings):
 
 class AppSettings(BaseSettings):
     CONFIG_FILE: Path = Path('evolver.yml')  # in current directory
+    LOAD_FROM_CONFIG_ON_STARTUP: bool = True
```

Comment:
I added this as I imagine this will want to be configurable and explicit such that it doesn't silently not load from config when the user might expect it to.